tests/mpi/periodicity_01.with_petsc=true.mpirun=3.output.2

```diff
@@ -0,0 +1,163 @@
+
+Testing for dim=2
+
+Cycle 0:
+DEAL::Starting value 0.00436405
```

Comment:
ok, i will keep it as it is now (second output file), if I understood you correctly.